### ISSUE TITLE
Module seems to break ~/error.vue functionality

### ISSUE THREAD

**Issue by tomsdob** [Baroshem/nuxt-security#185]
### Version

nuxt-security: ^0.14.2
nuxt: ^3.6.5

### Reproduction Link
https://stackblitz.com/edit/github-nmhzwq-j1jkla

### Steps to reproduce
Open the reproduction link, to see the custom error page working - comment out `nuxt-security` module from `nuxt.config.ts`.
### What is Expected?
The custom `~/error.vue` page is loaded.
### What is actually happening?
When using `nuxt-security` module, the custom `~/error.vue` page is not loaded.
### Additional info
When trying to reproduce the issue in StackBlitz, the app does not load anything at all after adding `nuxt-security` module. Locally the app works, but the custom `~/error.vue` page is shown only with 404 errors, for example error with `statusCode` 400 or 500 shows the default Nuxt error page with the following:
```
400
Bad Request

at createError (./node_modules/h3/dist/index.mjs:127:15)
at Object.handler (./.nuxt/dev/index.mjs:1258:17)
at Object.handler (./node_modules/h3/dist/index.mjs:1284:31)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
at async toNodeHandle (./node_modules/h3/dist/index.mjs:1359:7)
at async Object.ufetch [as localFetch] (./node_modules/unenv/runtime/fetch/index.mjs:9:17)
at async Object.errorhandler [as onError] (./.nuxt/dev/index.mjs:898:30)
at async Server.toNodeHandle (./node_modules/h3/dist/index.mjs:1366:9)
```

**Comment by Baroshem** [comment]
I just released the 0.14.3 version that should fix your issue. I tried it both locally and on Stackblitz and is seems to be working right now. Please let me know if that helps :)

**Comment by tomsdob** [comment]
@Baroshem thanks, the issue has been fixed!
